tutorial/source/tracking_1d_em.ipynb

```diff
@@ -245,27 +248,29 @@
     "                               ])\n",
     "    \n",
     "    exist_dist_probs = torch.stack([exist_dist_probs[i] for i in ids])\n",
-    "    exist_dist_probs = torch.cat([exist_dist_probs, \n",
+    "    assign_dist_probs = torch.stack([assign_dist_probs[i] for i in ids]).permute(1,2,0)\n",
```

Comment:
This logic is complex and slow. Let's instead recompute `assign_probs` and `exists_probs` from scratch after merging.